README.md

```diff
@@ -40,20 +40,36 @@ print('3:', test.evaluate(lambda n: n**2), 'wrong')      # wrong
 test.save('dobro.test')
  ```
 
- - After the users executed their tests and submit the solution you could run the table.
+ - After the users executed their tests and submit the solution you could run the table (with sort options).
```

Comment:
After the submission the solutions, it is possible to show the table with sort options.